### PR TITLE
json files included

### PR DESCRIPTION
**Pull request by prernapriyali**
building a library using javascript html and css

### CODE REVIEW

File: index.js
Comment (christianotieno) [inline index.js:1]: ## Section Specific Requirement.
- [ ] Do you mind keep the application logic in this module separate from DOM manipulation tasks just as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright? 

**We call this `Separation of Concerns`. An idea that each module or layer in an application should only be responsible for one thing and should not contain code that deals with other things. Separating concerns reduces code complexity by breaking a large application down into many smaller units of encapsulated functionality.**

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @prernapriyali 👋🏾 

Congratulations on the hard work with the library project. The code functions well and meets most of the requirements set. The repo is also decent and complies with the allowed standards. Keep it up.

Below I have highlighted some issues that might require you to pay attention to before this gets approved. The changes touch on modularising your code, have a look 👇🏾 

### Happy coding 💻 
#### Christian Otieno (CR)

File: index.js
Comment (prernapriyali) [reply index.js:1]: I do understand the concern you are trying to explain I completed the project as it was simple and short but if its needed I will change and if I can continue as I have done then it will be great at this point. will try to follow the concern from the next milestone.

File: index.js
Comment (teekaytech) [inline index.js:61]: Always strive to understand linter warnings and errors. This will ensure that you understand and probably find a way to fix the error, and not just disable it.

The `no-use-before-define` rule is eslint's way of telling you not to invoke a function before it is declared. In other words, moving the `checkBox` and `deleteBook` function up before the `createBookCard` function will fix this warning/error.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @prernapriyali  :wave:

__I must commend your efforts so far in this project. However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Update your Readme:
  - Add a screenshot of your project
  - Provide a live demo link
  - Fix the `Issues page` link: it's broken at the moment.
- [ ] Since we're here, kindly change the title of your PR. `json files included` is not a suitable name for a PR. Use a more descriptive title. 
- [ ] Check my inline comment below and fix it as appropriate.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hi @prernapriyali :hand: 
Awesome job on this project. Also, great job implementing the requested changes.
Good luck with the next project.

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**
